src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/StatusDropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { useField } from 'formik';
+import { DropdownField } from '../../../../../shared';
+
+const dropdownItems = [
+  { key: 'inProgress', value: 'In progress' },
+  { key: 'closed', value: 'Closed' },
+  { key: 'resolved', value: 'Resolved' },
```

Comment:
> This shows the status of the bug: Resolved, Unresolved.

As per Design docs, its Resolved or Unresolved but currently I see the below options, was this changed later on ? If yes then closed and resolved seems confusing.


<img width="283" alt="image" src="https://github.com/openshift/hac-dev/assets/9964343/21ee0e4a-3922-48e3-b150-a57f35f67ad3">